util/cosmos/iscn/query.js

```diff
@@ -45,6 +45,10 @@ export async function getISCNTransferInfo(txHash, opt) {
   // Not LIKE transfer, continue parse ISCN transaction payload
   parsed.tx.body.messages.forEach((m, index) => {
     const data = m.value.record;
+    let iscnVersion;
+    if (data.contentMetadata) {
+      iscnVersion = data.contentMetadata.version;
+    }
```

Comment:
Should use `recordVersion` which is set by chain, instead of the one in contentMetadata

Reply:
ok!